README.md

```diff
@@ -102,6 +106,11 @@ Configuration scripts can either take user input or can have environment variabl
         - Toggle GitHub integration (login). Accepts `true` or `false`
     - `RHDH_GITLAB_INTEGRATION`
         - Toggle GitLab integration (login). Accepts `true` or `false`
+    - `LIGHTSPEED_MODEL_URL`
+        - Target model URL for lightspeed plugin
+    - `LIGHTSPEED_API_TOKEN`
+        - API token for lightspeed plugin model service
+3. Run `source private.env` to set all set environment variables within `private.env`
```

Comment:
I believe with new changes from the commit below we can remove this line:
https://github.com/redhat-ai-dev/ai-rhdh-installer/pull/34

We should also include these env vars as part of the auto create for a private.env file for the user:
https://github.com/redhat-ai-dev/ai-rhdh-installer/blob/main/scripts/store-private-env.sh#L5

Reply:
`LIGHTSPEED_INTEGRATION` would also do well in the list of env vars as well IMO, I believe in our manual testing we thought it'd be best to have all of the env vars that can be set able to be preset in a env file for example

Reply:
done